pyrokinetics/equilibrium/equilibrium.py

```diff
@@ -347,8 +353,10 @@ def __init__(
 
         # Check that floats are valid
         psi_lcfs = float(psi_lcfs) * cocos_factors["PSI"] * eq_units["psi"]
-        if psi_lcfs < psi[0]:
-            raise ValueError("psi_lcfs should be greater than psi[0].")
+        if Ip_sign * psi_lcfs < Ip_sign * psi[0]:
+            raise ValueError(
+                f"{Ip_sign} * psi_lcfs should be greater than {Ip_sign} * psi[0]."
```

Comment:
Maybe something like `"psi_lcfs should be greater than psi[0] when I_p > 0, and vice versa when I_p < 0"`?